### ISSUE TITLE
[Bug]: Node rebooting from receiving invalid packet

### ISSUE THREAD

**Issue by liquidraver** [meshtastic/firmware#5508]
### Category

Other

### Hardware

T-Echo, Station G2

### Firmware Version

2.5.15

### Description

Opening a new bugreport, because I' not sure it's related to #5458 or #5498.
My T-Echo and my Station G2 is randomly rebooting, they are never alive for 24hrs.
The reboot happens usually after some hours uptime when I make a traceroute.

### Relevant log output

```Shell
.[36m.[0m..[34m.DEBUG .[0m| 20:08:21 15484 [RadioIf] .[34m.Started Tx (id=0x3ef60d7e fr=0xac to=0x90, WantAck=0, HopLim=0 Ch=0x8 encrypted rxtime=1733339301 priority=120).[0m
.[36m.[0m..[34m.DEBUG .[0m| 20:08:21 15484 [RadioIf] .[34m.Packet TX: 452ms.[0m
.[36m.[0m..[34m.DEBUG .[0m| 20:08:22 15484 [RadioIf] .[34m.Completed sending (id=0x3ef60d7e fr=0xac to=0x90, WantAck=0, HopLim=0 Ch=0x8 encrypted rxtime=1733339301 priority=120).[0m
.[36m.[0m..[34m.DEBUG .[0m| 20:08:28 15490 [Power] .[34m.Battery: usbPower=1, isCharging=1, batMv=4554, batPct=100.[0m
.[36m.[0m..[34m.DEBUG .[0m| 20:08:29 15491 [RadioIf] .[34m.Lora RX (id=0xc101d05c fr=0x90 to=0xac, WantAck=1, HopLim=2 Ch=0x8 encrypted rxSNR=6.5 rxRSSI=-38 hopStart=2).[0m
.[36m.[0m..[34m.DEBUG .[0m| 20:08:29 15491 [RadioIf] .[34m.Packet RX: 460ms.[0m
.[36m.[0m..[34m.DEBUG .[0m| 20:08:29 15491 [Router] .[34m.Found existing packet record for fr=0xSTATIONG2TRACEDNODE,to=0xTECHOWHOSTARTEDTHETRACE,id=0xc101d05c.[0m
.[36m.[0m..[34m.DEBUG .[0m| 20:08:29 15491 [Router] .[34m.Add packet record (id=0xc101d05c fr=0x90 to=0xac, WantAck=1, HopLim=2 Ch=0x8 encrypted rxSNR=6.5 rxRSSI=-38 hopStart=2).[0m
.[36m.[0m..[34m.DEBUG .[0m| 20:08:29 15491 [Router] .[34m.Ignore dupe incoming msg (id=0xc101d05c fr=0x90 to=0xac, WantAck=1, HopLim=2 Ch=0x8 encrypted rxSNR=6.5 rxRSSI=-38 hopStart=2).[0m
.[36m.[0m..[34m.DEBUG .[0m| 20:08:29 15492 [Router] .[34m.cancelSending id=0xc101d05c, removed=0.[0m
.[36m.[0m..[34m.DEBUG .[0m| 20:08:29 15492 [Router] .[34m.Repeated reliable tx.[0m
.[36m.[0m..[34m.DEBUG .[0m| 20:08:29 15492 [Router] .[34m.Partially randomized packet id 1475768703.[0m
.[36m.[0m..[34m.DEBUG .[0m| 20:08:29 15492 [Router] .[34m.Add packet record (id=0x57f6717f fr=0xac to=0x90, WantAck=0, HopLim=0 Ch=0x8 Portnum=5 requestId=c101d05c rxtime=1733339309 priority=120).[0m
.[31m.[0m..[31m.ERROR .[0m| 20:08:29 15492 [Router] .[31m.Invalid channel index 8 > 8, malformed packet received?.[0m
```

**Comment by GUVWAF** [comment]
Thanks for the report, I've identified why this happens and provided a temporary fix in #5509.

**Comment by GUVWAF** [comment]
Fixed by #5509.
